### DISCUSSION TITLE
Trello

### DISCUSSION THREAD

**Discussion by haribo841** [Projekt-Inzynierski-AK-AS-FD/Projekt-Inzynierski#12] [Q&A] [answered]
Potrzebuję pomocy z odpowiedziami na pytania. Trzeba by podzielić jakoś pracę. Pomożecie?
https://trello.com/b/y0GW2Max/green

**Comment by astolowska** [comment, marked as answer]
Jasne, ja pomogę. Chętnie omówię podział.
